common.sh

```diff
@@ -63,22 +63,6 @@ export SSH="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o C
 # Verify requisites/permissions
 # Connect to system libvirt
 export LIBVIRT_DEFAULT_URI=qemu:///system
-if [ "$USER" != "root" -a "${XDG_RUNTIME_DIR:-}" == "/run/user/0" ] ; then
-    echo "Please use a non-root user, WITH a login shell (e.g. su - USER)"
-    exit 1
-fi
-
-# Check if root user
-if [[ $EUID -eq 0 ]]; then
-   echo "This script must be executed as regular user"
-   exit 1
-fi
-
-# Check if sudo privileges without password
-if ! sudo -n uptime &> /dev/null ; then
```

Comment:
We should probably leave the sudo check, as it's needed for non-root users, and it will still work fine when run as root?

Reply:
Right, sudo would work with root too